serving/src/main/java/ai/djl/serving/http/KServeRequestHandler.java

```diff
@@ -258,4 +476,125 @@ private void onException(Exception ex, ChannelHandlerContext ctx) {
 
         NettyUtils.sendJsonResponse(ctx, content, status);
     }
+
+    void onModelException(Throwable t, ChannelHandlerContext ctx) {
+        HttpResponseStatus status;
+        if (t instanceof TranslateException) {
+            status = HttpResponseStatus.BAD_REQUEST;
+        } else if (t instanceof WlmException) {
+            logger.warn(t.getMessage(), t);
+            status = HttpResponseStatus.SERVICE_UNAVAILABLE;
+        } else {
+            logger.warn("Unexpected error", t);
+            status = HttpResponseStatus.INTERNAL_SERVER_ERROR;
+        }
+        if (ctx != null) {
+            NettyUtils.sendError(ctx, status, t);
+        }
+    }
+
+
+    public static final class KserveIO {
```

Comment:
We should never write two top level class in single file.

All other classes must be inner class.